Resources/Prototypes/_LostParadise/Loadouts/Jobs/security.yml

```
@@ -499,25 +499,22 @@
   items:
     - LPPWeaponRevolverJudge
 
+#Пульки
```

yamllint

[warning] missing starting space in comment

(comments)